tests/javaobject/PdxTests/PdxType.java

```diff
@@ -126,20 +127,21 @@ public void init()
     //m_dateTime = new Date( System.currentTimeMillis());
     
     m_int16Array = new short[] { 0x2332, 0x4545 };
-    m_uint16Array = new short[] { 0x3243, 0x3232 };
+    m_uint16Array = new short[] { (short)0xa243, (short)0xffff };
```

Comment:
These look like MAX_* constants, are there named constants for them anywhere?

Reply:
These java variables map to unsigned types on the C# side, where we do use the MAX* constants. I guess you could use whatever the macro for MIN_int16 (or whatever it's called) on the Java side, but that may be confusing.